.github/workflows/test-os.yml

```diff
@@ -48,11 +48,37 @@ jobs:
         name: Install gotestsum
         run: |
           go install gotest.tools/gotestsum@${{ env.GOTESTSUM_VERSION }}
+      -
+        name: Install buildkitd.exe binary
+        run: |
+          go install .\cmd\buildkitd
+      -
+        name: Install containerd.exe binary
+        run: |
+          $goModFileContent = Get-Content go.mod -Raw
+          $pattern = '(?<=github\.com/containerd/containerd v)[\d\.]+'
+          $version = [regex]::Match($goModFileContent, $pattern).Value
+          mkdir -p .\bin\installs -f
+          cd .\bin\installs
+          git clone https://github.com/containerd/containerd.git
```

Comment:
Eventually, this would need to be done with the Dockerfile. Either directly on windows or cross-compiled on linux. Eg. freebsd task in this file does the cross-compile.

Reply:
Good call out. After this, we will need to explore buildkitd running in the container (WCOW).

Reply:
Yes and same for distribution binaries, can look at this in follow-up :+1: